mail/mail/doctype/email_message/email_message.py

```diff
@@ -143,8 +152,8 @@ def get_message_ids(account: str, thread_ids: list[str], mailbox_id: str | None
 		validate_permission_for_account(account)
 
 		filters = {"account": account, "thread_id": ["in", thread_ids], "destroyed": 0}
-		if mailbox_id:
-			filters["mailbox_id"] = mailbox_id
+		if mailbox_id_filter:
+			filters["mailbox_id"] = mailbox_id_filter
```

Comment:
```suggestion
			filters["mailbox_id"] = ["in", mailbox_id_filter] if isinstance(mailbox_id_filter, list) else mailbox_id_filter
```

`mailbox_id_filter` can be `["a", "b"]`

Reply:
I'm directly passing the filter as a list here: https://github.com/frappe/mail/blob/3395004539a5d96ffd4acc94bf005ad42845027b/mail/api/mail.py#L470

For `["a", "b"]`, you'd have to pass `["in", ["a", "b"]]`.

Currently, this method is only being used by client APIs, so should be fine IG.